lib/realtime/timepoint_status.ex

```diff
@@ -28,7 +28,7 @@ defmodule Realtime.TimepointStatus do
     {past_stop_times, future_stop_times} = Enum.split_while(stop_times, &(&1.stop_id != stop_id))
 
     case Enum.find(future_stop_times, &StopTime.is_timepoint?(&1)) do
-      %StopTime{timepoint_id: next_timepoint_id} ->
+      %StopTime{timepoint: %Timepoint{id: next_timepoint_id}} ->
```

Comment:
(This is an example of a place where it'd be easier if most things reference the id directly when they don't need the name.)